# a header, used for interpreting the poses
std_msgs/Header header

# the information of tag detected 
TagDetection_msg[] tags_information